README.md

```diff
@@ -18,13 +18,31 @@ other programming languages as well as you do not have to take care of downloadi
 $ npm install @zazuko/rdf-vocabularies
 ```
 
+### Using in browser
+
+The package can be used in browser albeit it needs a bundler such as webpack and additional steps to configure it:
+
+* Enable [dynamic imports](https://medium.com/front-end-weekly/webpack-and-dynamic-imports-doing-it-right-72549ff49234). 
+  In webpack it is done with [@babel/plugin-syntax-dynamic-import](https://babeljs.io/docs/en/babel-plugin-syntax-dynamic-import)
+* Extend the bundler setup to have it load the contents of vocabulary files (all n-triples). In
+  In webpack it can be done with [`raw-loader`](https://github.com/webpack-contrib/raw-loader):
+
+      module: {
+        rules: [
+          {
+            test: /\.nq$/,
+            use: ['raw-loader']
+          }
+        ]
+      }
```

Comment:
Since all n-triples files take quite some space, it might make sense for some of this lib users to decide to only include some of the prefixes. For some projects that only use prefixes X, Y and Z, bundling all of them would be a waste of space regarding bundle size.

What do you think about adding an example of how to do this?

Reply:
> What do you think about adding an example of how to do this?

Actually, I don't know how and if this would be possible.

BUT, none of the datasets get bundled in the main bundle. Using dynamic `import()`, bundlers will put them in separate bundles. They will only be fetched when necessary, each one individually.

That said, I will actually mention this and suggest always using the `only` parameter so that unnecessary requests are avoided.